blueprints/incredible-squaring/src/main.rs

```diff
@@ -23,50 +22,63 @@ use gadget_sdk::{
     },
     tx,
 };
+use std::net::IpAddr;
 
 use incredible_squaring_blueprint::{self as blueprint, IncredibleSquaringTaskManager};
 
+use gadget_common::config::DebugLogger;
+use gadget_sdk::env::{AdditionalConfig, GadgetConfiguration};
 use gadget_sdk::events_watcher::tangle::TangleConfig;
+use gadget_sdk::keystore::BackendExt;
+use gadget_sdk::network::gossip::GossipHandle;
 use gadget_sdk::tangle_subxt::subxt;
+use gadget_sdk::tangle_subxt::tangle_testnet_runtime::api::runtime_types::sp_core::ed25519;
 use lock_api::{GuardSend, RwLock};
 use std::sync::Arc;
-use gadget_common::config::DebugLogger;
-use gadget_sdk::env::GadgetConfiguration;
-use gadget_sdk::network::gossip::GossipHandle;
+use alloy_signer::k256::PublicKey;
+use sp_core::Pair;
 
 #[async_trait::async_trait]
 trait GadgetRunner {
```

Comment:
Traits should now be in the SDK I think from @shekohex last PR. If you pull main in.